tools/clang/tools/dxclib/dxc.cpp

```diff
@@ -125,7 +125,7 @@ class DxcContext {
 
 private:
   DxcOpts &m_Opts;
-  DxcDllSupport &m_dxcSupport;
+  SpecificDllLoader &m_dxcSupport;
```

Comment:
This will presumably need to become a `DllLoader` in the future in order to support the ext validation case, right? This is using a SpecificDllLoader for now since you're planning to make those changes separately?